### DISCUSSION TITLE
有关fast retransmit的问题

### DISCUSSION THREAD

**Discussion by novada0** [SUSTech-CS305-Fall22/CS305-Project-Skeleton#41] [Q&A]
stop-wait不会出现duplicate acks的情况，所以我们是不是应该一次发多个pkt呢?stop-wait测试样例能通过，所以一次发多个pkt是不是optimization部分的呢?

**Comment by TANSixu** [comment]
Hi, 实现一次发多个pkt时基础要求的部分噢（详见文档congestion control以及窗口 部分）。congestion control部分会以同学们自己绘制的窗口变化图来检查。
